chart/hyrax/README.md

```diff
@@ -53,22 +54,35 @@ The chart populates the following environment variables:
 
 ## With an external SolrCloud
 
+By default, this chart deploys and manages an internal SolrCloud deployment,
+complete with its own ZooKeeper service.
+
 To use an existing or externally managed SolrCloud, use the chart values:
 
-  - `solr.enabled`: true
-  - `solr.externalSolrHost`: "mySolr.hostname.example.com"
-  - `solr.externalSolrUser`: "admin"
-  - `solr.externalSolrPassword`: "the_admin_password"
-  - `solr.externalSolrCollection`: "a_collection_name"
+  - `solr.enabled`: false
+  - `externalSolrHost`: "mySolr.hostname.example.com"
+  - `externalSolrUser`: "admin"
+  - `externalSolrPassword`: "the_admin_password"
+  - `externalSolrCollection`: "a_collection_name"
 
-By default, the chart will attempt to upload a ConfigSet for Hyrax matching the
-deployment's `hyrax.fullname` and assign it to the collection. This is achieved
-through the Solr Collections API, and authentication _must_ be enabled in the
-external Solr for this to work.
+The chart will attempt to upload a ConfigSet for Hyrax matching the deployment's
+"fullname" and assign it to the collection. This is achieved through the Solr
+Collections API, and authentication _must_ be enabled in the external Solr for
+this to work.
 
 If you want to manage your ConfigSet manually, disable this behavior with
 `--set loadSolrConfigSet=false`.
 
+## With an external Fedora Commons Repository
+
+By default, this chart deploys a local Fedora Repository, backed by the
+application's Postgresql system.
+
+To use an existing or external `fcrepo` instance, use the chart values:
+
+  - `fcrepo.enabled`: false
+  - `externalFcrepoHost`: "myfedora.hostname.example.com"
```

Comment:
is the fcrepo api unauthenticated by default?

Reply:
that's right. or at least, the upstream chart we're using doesn't turn on authentication.

supporting auth would probably be a desirable feature for both charts, but i'd like to keep it out of scope here.